pkg/proxy/proxy.go

```diff
@@ -85,10 +85,14 @@ func labels(sso *apiv1.SSO, appName string) map[string]string {
 }
 
 func serviceAnnotations(sso *apiv1.SSO, appName string) map[string]string {
+	var expIngressAnnotation string = ingressClassAnnotations + ": " + ingressClass;
```

Comment:
You can replace this with `expIngressAnnotation := ngressClassAnnotations + ": " + ingressClass;`